src/components/RouterComponent.jsx

```diff
@@ -11,6 +12,7 @@ const RouterComponent = () => {
       {/*--------------------a page for room initialization---------------------*/}
       <Route path="/room/:roomId" element={<RoomContainer />} />
       <Route path="/404" element={<Error />} />
+      <Route path="/leaderboard" element={<Leaderboard />} />
```

Comment:
Since we're going to work more on the responsive app bar i understand not doing it right now but, right now the only way to get to the leaderboard is to type in this url directly, right?
so the 'leaderboard' button in the appbar doesn't redirect to here